### ISSUE TITLE
App "is from an unidentified developer"

### ISSUE THREAD

**Issue by tofi86** [josejuanqm/ECheck#1]
```
→ codesign -vv ECheck.app
ECheck.app: valid on disk
ECheck.app: satisfies its Designated Requirement
```

Signature looks good. But:

<img width="547" alt="bildschirmfoto 2017-01-23 um 19 48 27" src="https://cloud.githubusercontent.com/assets/365118/22217992/6059912c-e1a5-11e6-9875-a87e8f9bd37f.png">

German message says that the App can't be opened because it is from an unidentified developer.

**Comment by josejuanqm** [comment]
yes, unfortunately i don't have the necessary privileges to sign the app (i don't pay for apple developer macOS program)

you can still open it by right clicking and choosing open.

Thanks for the feedback!
